### PR TITLE
Greenkeeper/initial

### PR DESCRIPTION
**Pull request by tzvipm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ This repository uses [lerna](https://github.com/lerna/lerna) to manage it's pack
...
 | `react-shopify-app-route-propagator` | [README](packages/react-shopify-app-route-propagator/README.md)                          | [![npm version](https://badge.fury.io/js/%40shopify%2Freact-shopify-app-route-propagator.svg)](https://badge.fury.io/js/%40shopify%react-shopify-app-route-propagator) |
-| `react-shortcuts`                    | [README](packages/react-shortcuts/README.md)                                             | [![npm version](https://badge.fury.io/js/%40shopify%2Freact-shortcuts.svg)](https://badge.fury.io/js/%40shopify%2Freact-html)                                          |
+| `react-shortcuts`                    | [README](packages/react-shortcuts/README.md)                                             | [![npm version](https://badge.fury.io/js/%40shopify%2Freact-shortcuts.svg)](https://badge.fury.io/js/%40shopify%2Freact-html) [![Greenkeeper badge](https://badges.greenkeeper.io/Shopify/quilt.svg)](https://greenkeeper.io/)                                          |
```
Comment (tzvipm) [inline README.md]: this is in the wrong spot.

File: greenkeeper.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        "packages/koa-metrics/package.json",
+        "packages/koa-shopify-auth/package.json",
+        "packages/koa-shopify-graphql-proxy/package.json",
+        "packages/react-compose/package.json",
+        "packages/react-html/package.json",
+        "packages/react-preconnect/package.json",
+        "packages/react-serialize/package.json",
+        "packages/react-shopify-app-route-propagator/package.json",
+        "packages/react-shortcuts/package.json",
+        "packages/with-env/package.json"
```
Comment (tzvipm) [inline greenkeeper.json]: I wonder if we'll need to update this on each new package?

File: packages/jest-mock-router/package.json [hunk 3]
```diff
@@ -23,10 +23,10 @@
   },
   "homepage": "https://github.com/Shopify/quilt/blob/master/packages/jest-mock-router/README.md",
   "devDependencies": {
-    "@types/react-router": "^3.0.0",
+    "@types/react-router": "^4.0.27",
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
-    "react-router": "^3.2.0",
-    "typescript": "~2.7.2"
+    "react-router": "^4.3.1",
```
Comment (tzvipm) [inline packages/jest-mock-router/package.json]: :-1:

**Comment by tzvipm** [comment]
I'll follow up with the lockfile in a separate PR: https://github.com/Shopify/quilt/issues/162

**Comment by tzvipm** [comment]
@TheMallen with the new linting config, vscode and the `yarn lint` command have some conflicts. If you have any ideas as to why that might be, please let me know.

**Comment by tzvipm** [comment]
I figured it out; once the packages for eslint have been upgrades, I just had to restart vs code in order for the changes to apply properly.

**Comment by tzvipm** [comment]
- [x] tests passing
- [x] linting passing
- [x] build passing

File: .nvmrc [hunk 4]
```diff
@@ -1 +1 @@
-v8.9.4
+v8.10.0
```
Comment (tsov) [inline .nvmrc]: Do we need any additional setup for travis? Or will it base it of the `.nvmrc` ?

Comment (tzvipm) [reply .nvmrc]: Travis uses the `nvmrc` :+1:

Comment (tsov) [reply .nvmrc]: 👌

**Comment by tsov** [comment]
:shipit:

**Pull request merged**